pkg/network/master/vnids.go

```diff
@@ -80,7 +80,7 @@ func (vmap *masterVNIDMap) isAdminNamespace(nsName string) bool {
 
 func (vmap *masterVNIDMap) markAllocatedNetID(netid uint32) error {
 	// Skip GlobalVNID, not part of netID allocation range
-	if netid == network.GlobalVNID {
+	if netid == network.GlobalVNID || netid == network.ControlPlaneVNID {
```

Comment:
hm... we were previously using VNID 2 for something as well, and might want to do that again later... I'd say just change this to `if netid < network.MinVNID {` (and then don't define `ControlPlaneVNID`)

Reply:
Fair enough, changed